packages/sit-onyx/src/styles/grid.scss

```diff
@@ -0,0 +1,57 @@
+//
+// Default Grid Variants
+//
+
+// prettier-ignore
+$gridVariants: (
+//  NAME     BREAKPOINT  COLUMNS  GUTTER  MARGIN
+    "2xs"    0px         4        16px    16px,
+    "xs"     576px       8        16px    16px,
+    "sm"     768px       8        24px    32px,
+    "md"     992px       12       24px    64px,
+    "lg"     1440px      16       32px    64px,
+    "xl"     1920px      16       32px    64px,
+);
+
+.onyx-grid {
+  display: grid;
+  grid-template-columns: repeat(var(--onyx-grid-columns), minmax(0, 1fr));
+  gap: var(--onyx-grid-gutter);
+  margin: var(--onyx-grid-margin);
+  max-width: var(--onyx-grid-max-width, none);
+}
+
+@each $name, $breakpoint, $columns, $gutter, $margin in $gridVariants {
+  @media screen and (min-width: $breakpoint) {
```

Comment:
Should we use a mixin for breakpoints? Similar to the one in `packages/vitepress-theme/src/mixins.scss`?
Would be useful when we implement components

Reply:
I tried, creating mixins in a loop is not possible in sass